lib/Sema/DerivedConformanceEquatableHashable.cpp

```diff
@@ -165,6 +165,12 @@ void diagnoseFailedDerivation(DeclContext *DC, NominalTypeDecl *nominal,
           nominal->getDeclaredInterfaceType());
     }
   }
+
+  if (auto *classDecl = dyn_cast<ClassDecl>(nominal)) {
+    ctx.Diags.diagnose(classDecl->getLoc(),
```

Comment:
Nit: You can use `Decl::diagnose` instead.

Reply:
Here I used same way to start diagnostic like it is above when we check enum and struct, to keep things consistent. Should we change it there too ?

Reply:
It's not a big deal.  Just something to look out for in the future.  In general, prefer `Decl::diagnose`.